lib/google_maps_service/apis/roads.rb

```diff
@@ -63,8 +63,9 @@ def snap_to_roads(path, interpolate: false)
     #         by the snap_to_roads function. You can pass up to 100 Place IDs.
     #
     # @return [Array] Array of speed limits.
-    def speed_limits(place_ids)
+    def speed_limits(place_ids, speed_unit='KPH')
       params = GoogleMapsService::Convert.as_list(place_ids).map { |place_id| ['placeId', place_id] }
+      params << ['units', speed_unit] if speed_unit == 'MPH'
```

Comment:
Same as before, to keep it similar to other methods
Could you please change to: `params[:units] = ...`?